cy-runner.yml

```diff
@@ -48,14 +40,17 @@ base:
     pageLoadTimeout: 45000
     browser: chrome
     chromeWebSecurity: false
-    sorry: false
   stateFiles:
     - .orders.json
 
 workspace:
   # If random a new name will be generated with prefix (ie b2b1234567)
   name: random
   prefix: affirmpayment
+  reserveAccount:
+    enabled: true
+    setup:
+      - vtex.orders-broadcast@0.x
```

Comment:
Everywhere we have hardTries as 1
Is this enough?

Reply:
yes